src/middleware/ChaosHandler.ts

```diff
@@ -185,7 +197,7 @@ export class ChaosHandler implements Middleware {
 	 * @param {string} requestURL - the URL for the request
 	 * @param {string} requestMethod - the API method for the request
 	 */
-	private setStatusCode(chaosHandlerOptions: ChaosHandlerOptions, requestURL: string, requestMethod: RequestMethod): boolean {
+	private getStatusCode(chaosHandlerOptions: ChaosHandlerOptions, requestURL: string, requestMethod: RequestMethod) {
```

Comment:
name getStatusCode means this function would return some status code... instead of updating it in the chaosHandlerOptions just return the status code from the function and assign the same in the callee function

Reply:
or let the name be setStatusCode.